src/components/learner-credit-management/BudgetCard-V2.jsx

```diff
@@ -7,58 +7,45 @@ import {
   Stack,
   Row,
   Col,
-  Breadcrumb,
 } from '@edx/paragon';
 
-import { useOfferRedemptions, useOfferSummary } from './data/hooks';
+import { useHistory } from 'react-router-dom';
+import { useOfferSummary } from './data/hooks';
 import LearnerCreditAggregateCards from './LearnerCreditAggregateCards';
-import LearnerCreditAllocationTable from './LearnerCreditAllocationTable';
-import { ROUTE_NAMES } from '../EnterpriseApp/data/constants';
+import { ROUTE_NAMES } from "../EnterpriseApp/data/constants";
 
 const BudgetCard = ({
   offer,
   enterpriseUUID,
   enterpriseSlug,
-  enableLearnerPortal,
 }) => {
   const {
     start,
     end,
   } = offer;
+  const history = useHistory();
 
   const {
     isLoading: isLoadingOfferSummary,
     offerSummary,
   } = useOfferSummary(enterpriseUUID, offer);
 
-  const {
-    isLoading: isLoadingOfferRedemptions,
-    offerRedemptions,
-    fetchOfferRedemptions,
-  } = useOfferRedemptions(enterpriseUUID, offer?.id);
-  const [detailPage, setDetailPage] = useState(false);
-  const [activeLabel, setActiveLabel] = useState('');
-  const links = [
-    { label: 'Budgets', url: `/${enterpriseSlug}/admin/${ROUTE_NAMES.learnerCredit}` },
-  ];
   const formattedStartDate = dayjs(start).format('MMMM D, YYYY');
   const formattedExpirationDate = dayjs(end).format('MMMM D, YYYY');
-  const navigateToBudgetRedemptions = (budgetType) => {
-    setDetailPage(true);
-    links.push({ label: budgetType, url: `/${enterpriseSlug}/admin/learner-credit` });
-    setActiveLabel(budgetType);
+  const navigateToBudgetRedemptions = (id) => {
+    history.push(`/${enterpriseSlug}/admin/${ROUTE_NAMES.learnerCredit}/${id}`);
   };
 
-  const renderActions = (budgetType) => (
+  const renderActions = (id) => (
     <Button
       data-testid="view-budget"
-      onClick={() => navigateToBudgetRedemptions(budgetType)}
+      onClick={() => navigateToBudgetRedemptions(id)}
```

Comment:
[suggestion] I would recommend treating this `Button` as a browser hyperlink with the react-router's `Link` component rather than relying on a JavaScript-only `history.push`. Users might want to open the budget details page in a new tab (e.g., right click and "Open in new tab", or `Cmd + click` on the button).

```jsx
<Button
  data-testid="view-budget"
  as={Link}
  to={`/${enterpriseSlug}/admin/${ROUTE_NAMES.learnerCredit}/${id}`}
>
  View budget
</Button>
```